### PR TITLE
Check current directory is git repo before branch

### PR DESCRIPTION
**Pull request by Luavis**
In fire, we can command this in un-proper directory. check it before start.

### CODE REVIEW

File: git-fire [hunk 1]
```diff
@@ -27,6 +27,11 @@ new_branch() {
 }
 
 fire() {
+	if [ ! -d .git ] && ! git rev-parse --git-dir > /dev/null 2>&1; then
+		echo "What are you doing? It is not git repo!!, hurry";
+		exit -1
```
Comment (stephenmathieson) [inline git-fire:32]: negative exit code?

Comment (Luavis) [inline git-fire:32]: It is improper situation to run git-fire in not git working space, isn't it?

Comment (CodeBrauer) [inline git-fire:32]: `exit -1` is just `exit 255`

Comment (qw3rtman) [inline git-fire:32]: `exit 1` is probably more conventional.

File: git-fire [hunk 2]
```diff
@@ -27,6 +27,11 @@ new_branch() {
 }
 
 fire() {
+	if [ ! -d .git ] && ! git rev-parse --git-dir > /dev/null 2>&1; then
+		echo "What are you doing? It is not git repo!!, hurry";
```
Comment (qw3rtman) [inline git-fire:31]: Change error message to `This is not a Git repo, hurry!`

**Pull request closed without merging**